src/Microsoft.Xaml.Behaviors.DesignTools/MetadataTableProvider.cs

```diff
@@ -12,16 +13,16 @@ namespace Microsoft.Xaml.Behaviors.DesignTools
 {
     internal class MetadataTableProvider : IProvideAttributeTable
     {
-        private AttributeTableBuilder _attributeTableBuilder;
+        private AttributeTableBuilder attributeTableBuilder;
```

Comment:
why remove the underline from the private variable?

Reply:
Restored the underscore.